torch/_inductor/select_algorithm.py

```diff
@@ -3620,9 +3622,12 @@ def key_of(node):
                 node.get_size(),
                 fallback=config.unbacked_symint_fallback,
             ),
-            *sizevars.size_hints(
-                node.get_stride(),
-                fallback=config.unbacked_symint_fallback,
+            tuple(
+                V.graph.sizevars.atomically_apply_size_hint(
+                    stride,
+                    fallback=config.unbacked_symint_fallback,
+                )
+                for stride in node.get_stride()
             ),
```

Comment:
Should keep the tuple unpacking

```suggestion
            *tuple(
                V.graph.sizevars.atomically_apply_size_hint(
                    stride,
                    fallback=config.unbacked_symint_fallback,
                )
                for stride in node.get_stride()
            ),
```

Reply:
thanks for catching this!